docs/UserGuide.md

```diff
@@ -508,17 +508,24 @@ Examples:
 
 #### 3.4.1 Scheduling: `schedule` (By: Alex) 
 
-##### 3.4.1.1 Viewing classes on a Timetable: `schedule view`  
+##### 3.4.1.1 Viewing classes on a Timetable: `**schedule view**`  
```

Comment:
When I did a run on the schedule command, it appears that `schedule mode/MODE date/DATE` is accepted but not `schedule view mode/MODE date/DATE`. Perhaps you could update the UG to reflect this, and remove `PREFIX_VIEW` from the parser package.

Reply:
Yea thanks alot!